workflow-templates/im-test-selenium-schedule.yml

```diff
@@ -0,0 +1,201 @@
+# Workflow Code: CosmicFalcon_v1   DO NOT REMOVE
+# Purpose:
+#    Runs Selenium C# tests in GitHub Actions on a schedule.
+#
+# Frequency:
+#    - Workflow template for running Selenium tests on a schedule. This workflow supports the UI Health Check feature
+#
+# Projects to use this Template with:
+#    - Testing projects that use Selenium C#.
+#
+# TODO Prerequisites:
+#    - Add Application Insights secret (APPLICATIONINSIGHTS_CONNECTION_STRING) to every environment you are testing.
+#    - Update your Selenium solution to export test results to Application Insights using the IM.ExportToAppInsights NuGet package. 
+#      Detailed setup guidance can be found here: https://github.com/im-practices/techhub/blob/main/docs/Azure/content/Health-Checks/Selenium-UI-Health-Check.md
+
+name: 🧪 Scheduled Selenium C# Tests
+run-name: 🧪 Run Scheduled Selenium C# Tests
```

Comment:
What are some of the use cases for running Selenium tests on a schedule?

Reply:
We schedule these Selenium tests primarily to perform periodic health checks and send the results to AUT App Insights to track UI issues in production. This helps teams detect issues quickly and monitor uptime effectively.

Reply:
Currently, four teams are automating their UI testing with Selenium C#. As these teams—and potentially others—look to implement or update GitHub Actions for PR reviews, health check validations, and similar needs, these workflows will be a helpful resource.